test/env.jl

```diff
@@ -122,6 +122,38 @@ if Sys.iswindows()
     end
 end
 
+@testset "get_bool_env" begin
```

Comment:
Should test also error path, perhaps with a [mispelled value](https://github.com/JuliaLang/Pkg.jl/pull/2858#issue-1062953232).